packages/test/test-service-load/src/utils.ts

```diff
@@ -38,11 +38,37 @@ import { ILoadTestConfig, ITestConfig } from "./testConfigFile";
 const packageName = `${pkgName}@${pkgVersion}`;
 
 class FileLogger extends TelemetryLogger implements ITelemetryBufferedLogger {
+	private static readonly loggerP = new LazyPromise<FileLogger>(async () => {
```

Comment:
this is the core of the change. we now pub the singleton inside this class, and make the constructor  private. this forces all uses to go through the static createLogger and flushLogger on this class

Reply:
for usage outside this file, we export the `createLogger` function below